src/theory/quantifiers/ce_guided_single_inv_sol.cpp

```diff
@@ -300,7 +300,7 @@ bool CegConjectureSingleInvSol::getAssignEquality( Node eq, std::vector< Node >&
   TypeNode tn = eq[0].getType();
   if( tn.isInteger() || tn.isReal() ){
     std::map< Node, Node > msum;
-    if( QuantArith::getMonomialSumLit( eq, msum ) ){
+    if( ArithMSum::getMonomialSumLit( eq, msum ) ){
```

Comment:
Format?

Reply:
It's dead code, perhaps I should remove.